### PR TITLE
I create realisation LinkedList

### PR DESCRIPTION
**Pull request by Swyatik19**

### CODE REVIEW

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 1]
```diff
@@ -1,47 +1,154 @@
...
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    public MyLinkedList() {
+        size = 0;
+    }
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/MyLinkedList.java:13]: Why do we need this constructor?

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 2]
```diff
@@ -1,47 +1,154 @@
...
+        } else {
+            lastNode.next = newNode;
+        }
+        size++;
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
+
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/MyLinkedList.java:31]: redundant empty line

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 3]
```diff
@@ -1,47 +1,154 @@
...
+        size--;
+        return value;
     }
 
     @Override
-    public boolean remove(T object) {
+    public boolean remove(T t) {
+        Node<T> node = head;
+        for (int i = 0; i < size; i++) {
+            if (Objects.equals(t, node.item)) {
```
Comment (MaliukDaria) [inline src/main/java/core/basesyntax/MyLinkedList.java:102]: [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/linked-list.html): Don’t use class Objects.

**Review by MaliukDaria — CHANGES_REQUESTED** [review]
Before you create PR with your homework, you need to go through the checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist points are corrected

File: src/main/java/core/basesyntax/MyLinkedList.java (hunk 1 above)
Comment (Swyatik19) [reply src/main/java/core/basesyntax/MyLinkedList.java:13]: Привіт я не розумію що в цьому коментарі зміинити?

**Review by MaliukDaria — APPROVED** [review]

**Pull request closed without merging**